packages/simple-markdown/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@khanacademy/simple-markdown",
+  "version": "0.8.0",
+  "description": "Javascript markdown parsing, made simple",
+  "main": "simple-markdown.js",
+  "types": "simple-markdown.d.ts",
+  "scripts": {
+    "build": "make build",
+    "test": "make test",
+    "prepublishOnly": "make build",
+    "size": "size-limit"
+  },
+  "keywords": [
+    "markdown",
+    "simple",
+    "react",
+    "html",
+    "ast",
+    "markup",
+    "text processing",
+    "extension",
+    "flow",
+    "typescript"
+  ],
+  "author": "Khan Academy",
+  "license": "MIT",
+  "files": [
+    "index.js",
+    "simple-markdown.js",
+    "simple-markdown.d.ts",
+    "simple-markdown.min.js",
+    "README.md"
+  ],
+  "dependencies": {
+    "@types/react": ">=16.0.0"
```

Comment:
Can you move this to be a `peerDependency` (that way we won't bundle it with this package).  Then also make it a `devDependency` so it's installed when we're working on the library in this repo.